flake.nix

```diff
@@ -7,6 +7,7 @@
     extra-trusted-public-keys = [
       "nix-cache.minaprotocol.org:fdcuDzmnM0Kbf7yU4yywBuUEJWClySc1WIF6t6Mm8h4="
       "nix-cache.minaprotocol.org:D3B1W+V7ND1Fmfii8EhbAbF1JXoe2Ct4N34OKChwk2c="
+      "mina-nix-cache-1:djtioLfv2oxuK2lqPUgmZbf8bY8sK/BnYZCU2iU5Q10="
```

Comment:
It's not clear who has access to this key, where it is, what it's used for, etc. Could you add those details somewhere? I assume that this file doesn't allow comments, perhaps you should edit the commit message to include those details?

Reply:
I want to use this key in our CI, initiated talk with PLatform team about it.

I named the key `mina-nix-cache-1` because GCP bucket name is `mina-nix-cache` and this is a key that is generated specifically for use with the bucket.

I can ammend the commit message to mention CI if you'd like

Reply:
Updated the PR

Reply:
This PR leverages envs injected into Buildkite `generic` and `small` agents: https://github.com/MinaProtocol/mina-deployments/pull/22

The referenced PR has already been applied to enable this one.